docs/guides/errors.md

```diff
@@ -0,0 +1,156 @@
+
+---
+
+### NO_METHOD_MATCHED_ERROR
+**Message: Route resolved, but no path matched**
+#### Returned Status Code: `405`
+**Explanation:** This error occurs when the current document has an endpoint with the requested URL, but the specified Verb is not listed.
+
+##### Example
+
+```yaml
+openapi: 3.0.1
+paths:
+  /pets:
+    get:
+      responses:
+        200:
+          description: Hey
+```
+
+`curl -X POST http://localhost:4010/pets`
+
+---
+
+### NO_SERVER_CONFIGURATION_PROVIDED_ERROR
+**Message: Route not resolved, no server configuration provided**
+#### Returned Status Code: `404`
+**Explanation:**
+
+## Validation errors
+
+This class of errors is returned when Prism is validating the request/response against the provided OpenAPI file.
+
+### UNPROCESSABLE_ENTITY
+**Message: Invalid request body payload**
+#### Returned Status Code: `422`
+**Explanation:** This error occurs when the current request has not passed the validation rules specified in the current OpenAPI file *and* the current resource is missing an error message (`422`, `400`, `default`). Note that this is an error _genered_ by Prism.
+
+The `detail` field contains further information on the error (whether it's on the body, the headers or the query string).
+
+---
+
+### NOT_ACCEPTABLE
+**Message: The server cannot produce a representation for your accept header**
+#### Returned Status Code: `406`
+**Explanation:** This error occurs when the current request has asked the response in a format that the current document is not able to produce.
+
+##### Example
+
+```yaml
+openapi: 3.0.2
+paths:
+  /todos:
+    get:
+      responses:
+        200:
+          description: Get Todo Items
+          examples:
+            text/plain: hello
+```
+
+`curl http://localhost:4010/todos -H "accept: application/json"`
+
+---
+
+### NOT_FOUND
+**Message: The server cannot find the requested content**
+#### Returned Status Code: `404`
+**Explanation:** This error occurs when the current request is asking for a specific status code that the document is not listing or it's asking for a specific example that does not exist in the current document
+
+---
+
+## Security errors
+
+This class of errors is returned when the current request is not satisfying the security requirements specified in the current resource
+
+### UNAUTHORIZED
+**Message: Invalid security scheme used**
+#### Returned Status Code: `401`
+**Explanation:** This error occurs when the security scheme for the current resource does not match the one that the one that the request being processed has provided.
+
+---
+
+### FORBIDDEN
+**Message: Invalid credentials used**
+#### Returned Status Code: `403`
+**Explanation:** This error occurs when the current request is using the correct security scheme, but its content results invalid. This might be an error in the decoding process (such as a not valid base64 payload)
```

Comment:
```suggestion
**Explanation:** This error occurs when the current request is using the correct security scheme, but its content was invalid. This might be an error in the decoding process (such as a not valid base64 payload)
```